### DISCUSSION TITLE
Start up sound, but not playing any notes

### DISCUSSION THREAD

**Discussion by ZappaJ** [cardonabits/haxo-hw#86] [Q&A]
Hello all,

I had the haxophone working, but then I used the documentation to change the default instrument to a baritone sax, number 67.  I then rebooted and it didn't make any sounds.  I reinstalled the latest haxophone raspberry pi image and installed it on my RP Zero.  I then rebooted.  It made the start up sound, but when I tried to play, there is no sound.  

Can anyone help me diagnose this?

**Comment by jcard0na** [comment]
Hi @ZappaJ,

Are you able to log into the haxophone to examine logs?  The three ways to do that are documented here: https://github.com/cardonabits/haxo-hw/blob/main/docs/login.md
